docs/application/dotnet/guides/security/dpm.md

```diff
@@ -0,0 +1,99 @@
+# Device Policy Management
+
+
+The Device Policy Management (DPM) framework supports enterprise applications by providing IT(Information Technology) administrator functions to create security-aware applications. These applications are useful in scenarios where IT administrators require rich control over employee devices.
```

Comment:
Please add space after IT......
IT (Information Technology)

Reply:
Done.